plugins/plugin-sass/plugin.js

```diff
@@ -0,0 +1,76 @@
+const execa = require('execa');
+const fs = require('fs');
+const path = require('path');
+
+const IMPORT_REGEX = /\@(use|import)\s*['"](.*?)['"]/g;
+
+function scanSassImports(fileContents, filePath) {
+  // TODO: Replace with matchAll once Node v10 is out of TLS.
+  // const allMatches = [...result.matchAll(new RegExp(HTML_JS_REGEX))];
+  const allMatches = [];
+  let match;
+  const regex = new RegExp(IMPORT_REGEX);
+  while ((match = regex.exec(fileContents))) {
+    allMatches.push(match);
+  }
+
+  return allMatches
+    .map((match) => match[2])
+    .filter((s) => s.trim())
+    .map((s) => {
+      if (!path.extname(s)) {
+        s += '.scss';
+      }
+      return path.resolve(path.dirname(filePath), s);
+    });
+}
+
+module.exports = function postcssPlugin(_, options) {
+  const importedByMap = new Map();
+
+  function addImportsToMap(filePath, sassImports) {
+    for (const imported of sassImports) {
+      const importedBy = importedByMap.get(imported);
+      if (importedBy) {
+        importedBy.add(filePath);
+      } else {
+        importedByMap.set(imported, new Set([filePath]));
+      }
+    }
+  }
+
+  return {
+    name: '@snowpack/plugin-sass',
+    resolve: {
+      input: ['.scss'],
```

Comment:
```suggestion
      input: ['.sass', '.scss'],
```

Reply:
Ah yea, you saw below that we assume scss. I didn't see anything on the docs site about `.sass` (everything says `.scss` more-or-less required). is that still a thing that we need to support?

Reply:
Yes `.sass` comes for free with any Sass implementation. I think you can try and resolve `.scss` first, but `.sass` should also be attempted as a fallback